TestFoundation/TestURLSession.swift

```diff
@@ -119,6 +120,56 @@ class TestURLSession : LoopbackServerTest {
         waitForExpectations(timeout: 12)
     }
     
+    func test_dataTaskWithHttpInputStream() {
+        func randomString(length: Int) -> String {
+            let letters : NSString = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+            let len = UInt32(letters.length)
+            
+            var randomString = ""
+            
+            for _ in 0 ..< length {
+                let rand = arc4random_uniform(len)
```

Comment:
This needs to be updated to use `Int.random(in: 0..<len)` as `arc4random_uniform` is not available on Linux

Reply:
@spevans fixed! Thanks!

Reply:
If we’re targetting only Swift 4.2 and higher (since I doubt this fix would be back-ported to, say, a 4.1.4 release), may I ask why not use the new random API as suggested by @spevans ?

Reply:
Oh, I miss the new Random API. Will fix it/